dashboard-api/docker-compose-common.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  cockroachdb:
+    image: cockroachdb/cockroach:v21.2.3
+    entrypoint: cockroach start-single-node --insecure --advertise-addr localhost
+    ports:
+      - "26257:26257"
+      - "8080:8080"
```

Comment:
How are we using port 8080? I thought cockroach only required port 26257?

Reply:
It exposes the cockroach dashboard that displays some metrics about the database.
I haven't found it useful to troubleshoot any issue, I'll remove this line for now 👌